### ISSUE TITLE
Debugger isn't available.

### ISSUE THREAD

**Issue by TimKrause2** [jupyterlab/jupyterlab#18146]
## Description

The **Restart Kernel and Debug** option is grayed out. The debugger can't be started.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/9def87fc-57d0-437b-80e5-0ac3e8989217" />

## Reproduce

I've tried a fresh python environment and reinstalled the latest packages with pip.

## Context

- Operating System and version: Linux Ubuntu 25.10
- Browser and version: Chrome Version 142.0.7444.59 (Official Build) (64-bit)
- JupyterLab version: 4.5.0

<details><summary>Troubleshoot Output</summary>
<pre>
$PATH:
	/home/tim/python3_venv/bin
	/home/tim/.vcpkg
	/home/tim/.local/bin
	/usr/local/sbin
	/usr/local/bin
	/usr/sbin
	/usr/bin
	/sbin
	/bin
	/usr/games
	/usr/local/games
	/snap/bin
	/snap/bin

sys.path:
	/home/tim/python3_venv/bin
	/usr/lib/python313.zip
	/usr/lib/python3.13
	/usr/lib/python3.13/lib-dynload
	/home/tim/python3_venv/lib/python3.13/site-packages

sys.executable:
	/home/tim/python3_venv/bin/python

sys.version:
	3.13.7 (main, Aug 20 2025, 22:17:40) [GCC 15.2.0]

platform.platform():
	Linux-6.17.0-6-generic-x86_64-with-glibc2.42

which -a jupyter:
	/home/tim/python3_venv/bin/jupyter

pip list:
	Package                   Version
	------------------------- -----------
	absl-py                   2.3.1
	anyio                     4.11.0
	argon2-cffi               25.1.0
	argon2-cffi-bindings      25.1.0
	arrow                     1.4.0
	asttokens                 3.0.1
	astunparse                1.6.3
	async-lru                 2.0.5
	attrs                     25.4.0
	babel                     2.17.0
	beautifulsoup4            4.14.2
	bleach                    6.3.0
	certifi                   2025.11.12
	cffi                      2.0.0
	charset-normalizer        3.4.4
	comm                      0.2.3
	debugpy                   1.8.17
	decorator                 5.2.1
	defusedxml                0.7.1
	executing                 2.2.1
	fastjsonschema            2.21.2
	flatbuffers               25.9.23
	fqdn                      1.5.1
	gast                      0.6.0
	google-pasta              0.2.0
	grpcio                    1.76.0
	h11                       0.16.0
	h5py                      3.15.1
	httpcore                  1.0.9
	httpx                     0.28.1
	idna                      3.11
	ipykernel                 7.1.0
	ipython                   9.7.0
	ipython_pygments_lexers   1.1.1
	ipywidgets                8.1.8
	isoduration               20.11.0
	jedi                      0.19.2
	Jinja2                    3.1.6
	json5                     0.12.1
	jsonpointer               3.0.0
	jsonschema                4.25.1
	jsonschema-specifications 2025.9.1
	jupyter                   1.1.1
	jupyter_client            8.6.3
	jupyter-console           6.6.3
	jupyter_core              5.9.1
	jupyter-events            0.12.0
	jupyter-lsp               2.3.0
	jupyter_server            2.17.0
	jupyter_server_terminals  0.5.3
	jupyterlab                4.5.0
	jupyterlab_pygments       0.3.0
	jupyterlab_server         2.28.0
	jupyterlab_widgets        3.0.16
	keras                     3.12.0
	lark                      1.3.1
	libclang                  18.1.1
	Markdown                  3.10
	markdown-it-py            4.0.0
	MarkupSafe                3.0.3
	matplotlib-inline         0.2.1
	mdurl                     0.1.2
	mistune                   3.1.4
	ml_dtypes                 0.5.4
	namex                     0.1.0
	nbclient                  0.10.2
	nbconvert                 7.16.6
	nbformat                  5.10.4
	nest-asyncio              1.6.0
	nodejs                    0.1.1
	notebook                  7.5.0
	notebook_shim             0.2.4
	numpy                     2.3.5
	opt_einsum                3.4.0
	optional-django           0.1.0
	optree                    0.18.0
	packaging                 25.0
	pandocfilters             1.5.1
	parso                     0.8.5
	pexpect                   4.9.0
	pillow                    12.0.0
	pip                       25.1.1
	platformdirs              4.5.0
	prometheus_client         0.23.1
	prompt_toolkit            3.0.52
	protobuf                  6.33.1
	psutil                    7.1.3
	ptyprocess                0.7.0
	pure_eval                 0.2.3
	pycparser                 2.23
	Pygments                  2.19.2
	python-dateutil           2.9.0.post0
	python-json-logger        4.0.0
	PyYAML                    6.0.3
	pyzmq                     27.1.0
	referencing               0.37.0
	requests                  2.32.5
	rfc3339-validator         0.1.4
	rfc3986-validator         0.1.1
	rfc3987-syntax            1.1.0
	rich                      14.2.0
	rpds-py                   0.29.0
	Send2Trash                1.8.3
	setuptools                80.9.0
	six                       1.17.0
	sniffio                   1.3.1
	soupsieve                 2.8
	stack-data                0.6.3
	tensorboard               2.20.0
	tensorboard-data-server   0.7.2
	tensorflow                2.20.0
	termcolor                 3.2.0
	terminado                 0.18.1
	tinycss2                  1.4.0
	tornado                   6.5.2
	traitlets                 5.14.3
	typing_extensions         4.15.0
	tzdata                    2025.2
	uri-template              1.3.0
	urllib3                   2.5.0
	wcwidth                   0.2.14
	webcolors                 25.10.0
	webencodings              0.5.1
	websocket-client          1.9.0
	Werkzeug                  3.1.3
	wheel                     0.45.1
	widgetsnbextension        4.0.15
	wrapt                     2.0.1
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>

[jupyter-lab.txt](https://github.com/user-attachments/files/23733499/jupyter-lab.txt)

</pre>
</details>

<details><summary>Browser Output</summary>

<pre>
See the screenshot.
</pre>
</details>

**Comment by TimKrause2** [comment]
I just realized that the Browser Output in the console isn't legible. So I thought I would make a better screen shot of the console output.

<img width="554" height="490" alt="Image" src="https://github.com/user-attachments/assets/0f69dcc5-600d-4c74-a614-407b757694ca" />

**Comment by krassowski** [comment]
What if you click on the bug icon in the toolbar?

**Comment by TimKrause2** [comment]
Oh. I didn't see that before.
That was it.
Thanks for your help.
